container-search/src/test/java/com/yahoo/search/grouping/result/GroupListTestCase.java

```diff
@@ -1,6 +1,7 @@
 // Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 package com.yahoo.search.grouping.result;
 
+import com.yahoo.search.Query;
```

Comment:
why was this lonely import added ? Back and forth ?

Reply:
Yep. Removed, thanks